example/src/main/java/org/apache/rocketmq/example/quickstart/Consumer.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) throws InterruptedException, MQClientExce
          * </pre>
          */
         // Uncomment the following line while debugging, namesrvAddr should be set to your local address
-//        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
+        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```

Comment:
If you've already uncommented the line, you'd better change the content in the comment

Reply:
thks.I missed that.